pytest_selenium/pytest_selenium.py

```diff
@@ -94,8 +94,12 @@ def driver_kwargs(request, capabilities, chrome_options, driver_args,
         driver_path=driver_path,
         firefox_options=firefox_options,
         firefox_profile=firefox_profile,
-        host=pytestconfig.getoption('host'),
-        port=pytestconfig.getoption('port'),
+        host=pytestconfig.getoption('host',
+                                    getattr(driver, 'HOST', None),
+                                    True),
+        port=pytestconfig.getoption('port',
+                                    getattr(driver, 'PORT', None),
+                                    True),
```

Comment:
Doing it this way means the `pytest_configure` hook isn't able to access these values, which it uses for adding to the metadata. I'm going to see if we can simply move this code into the hook.